openpype/hosts/max/plugins/create/create_render.py

```diff
@@ -31,3 +45,39 @@ def create(self, subset_name, instance_data, pre_create_data):
         RenderSettings().set_render_camera(sel_obj)
         # set output paths for rendering(mandatory for deadline)
         RenderSettings().render_output(container_name)
+
+    def get_instance_attr_defs(self):
+        return [
+            BoolDef("use_published",
+                    default=setting()["active"],
+                    label="Use Published Scene"),
+
+            NumberDef("priority",
+                      minimum=1,
+                      maximum=250,
+                      decimals=0,
+                      default=setting()["priority"],
+                      label="Priority"),
+
+            NumberDef("chunkSize",
+                      minimum=1,
+                      maximum=50,
+                      decimals=0,
+                      default=setting()["chunk_size"],
+                      label="Chunk Size"),
+
+            TextDef("group",
+                    default=setting()["group"],
+                    label="Group Name"),
+
+            TextDef("deadline_pool",
+                    default=setting()["deadline_pool"],
+                    label="Deadline Pool"),
+
+            TextDef("deadline_pool_secondary",
+                    default=setting()["deadline_pool_secondary"],
+                    label="Deadline Pool Secondary")
+        ]
```

Comment:
This should **really** be somewhere in the Deadline module and not on Create Render.

Reply:
Yes i know, but this is client's request so I need to put it there